examples/webgl_materials_envmaps_hdr.html

```diff
@@ -275,6 +278,23 @@
 
 					}
 
+
+					var side = standardMaterial.side;
+
+					switch ( params.side ) {
+
+						case 'front': side = THREE.FrontSide; break;
+						case 'back': side = THREE.BackSide; break;
+						case 'double': side = THREE.DoubleSide; break;
+	
+					}
+
+					if( standardMaterial.side !== side ) {
+						
+						standardMaterial.side = side;
+						standardMaterial.needsUpdate = true;
```

Comment:
yes, because gl_FrontFacing is only checked if the define DOUBLE_SIDED is set and defines only change their state when one recompiles the shaders.